docs/changelog/1236.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+fix typos in isolated.py that made it impossible to install package with requirements in pyproject.toml - by :user:`unmade`
```

Comment:
you don't have to add the issue, is auto generated, just stop as the by user 👌

Reply:
Oh...)
I removed link to issue